Remade Bridge Deck

Albens opened the pull request:
:cl: Albens
maptweak: Bridge Deck has undergone a refit. 
/:cl:

Bridge deck has been Refit.

<details><summary>Expanded Details.</summary>
<p>

I've refit Bridge Deck, touching the Head's Offices, the Secpoint, Bridge Safe Room, Aux EVA, Bridge Fore Maintenance, Disciplinary room, and of course, Bridge Itself.

Fancy new Desks for the CO and XO have been added.

Two new Areas, the CL's backroom, and Deliberation Room.

The Bridge Combined Arms Locker, and the Bridge Safe Room Firearm locker have had their inventories mostly retained, but shuffled into two Lockers.
I did trade 2 small eguns for 1 large Egun + belt. This was done to reflect the Bridge staffing levels, and I believe it might alleviate an Escalation of Force issue, caused by having a deficiency in "Suitable" firearms for Command and Command Support. 

In Short: Command has _just_ enough full sized Eguns on Bridge that they shouldn't feel the need to run to Earm to grab more.

Bridge Storage now has two lockers: Combined Arms Cabinet, and Bridge PPE Cabinet.

Bridge Safe Room is now drastically smaller, and unfit for Habitation for long periods of time, using my THERA Concept. 

There's an Atmospheric Support station in Bridge-Deck Fore Starboard Maintenance. It's like those dual-Pressure tanks that exist currently, but less useless. Engineering can set them up further if they want.

Bridge has been designed to minimize the two dominant Meta-strategies. 

Emagging onto a Locked-down Bridge now must be done through some of the Secured, Bolted Emergency Exits, or the Safe Room Auxiliary Door. This adds time, in the form of both Hacking and acquiring Tools.

Thermite-ing into a Command-level locker now takes marginally more time. The XO's locker is hard to get at discreetly, while the CO and SEA's locker require you to breach a regular wall, and then an R-wall. This adds time.

These two measures should hopefully reduce the occurrence of, and introduce some risk into the _lolEmag_ and _lolThermite_ Metas.

The Disciplinary Board Room now has a Deliberation Sub-Room. No more awkward whispering at the Officer's desk.

The CL's office now contains a backroom. Due to the lack of actual function to the CL as a job, I've used the LPA and some creative door/tint/light controls to give multi-function to the CL's office, designed to create a power imbalance between the CL and their "prey".

Personal Offices have had their disposals, photocopiers, and shredders removed, re-added on a case-by-case basis.
These served to clutter offices, and their function was extremely limited. The disposals where largely used for bombings or by mailing mistakes.

Bridge Storage and Bridge Meeting Room are now the only valid mailing locations on Bridge Deck. Trust me, you won't miss personal office disposals bins. 

Aux EVA has been expanded Port, adding in one more row of suits; a Command and Security Voidsuit have been added, to reflect updated staffing numbers.

A locked-down Bridge presents a fortress like-appearance, but has vulnerabilities all the same. 

If I remember any more changes, I'll update this!

Special thanks to:
Orelbon, Eckles, Sabira, Norodo, Mattatlas, Ondra, Sweedle, SierraKomodo, Xaytan, Fre3bie, BlueNexus, Lonefly, QuestioningMark, Flying_loulou, Roland410, and any others I forgot.
</p>
</details>
<!--

babydoll commented:
The map diff is gone so you're going to have to provide images

Chiquiii commented:
There are images though?

Albens commented:
All that _should_ be left is the code quality check, which- frankly- I'm not actually sure which part I need to correct.

Also the map picture got updated.

Albens commented:
All is good, I think. 

I've updated based on Feedback, and have updated the Public Hallway Lockdown floor decals I wasn't consistent on. The Bridge internal lockdown shutter follows the Bridge Lockdown floor decals, so it wasn't overlooked.

The rest of the post-PR feedback I've gotten has been based on personal Opinions regarding Intentional Changes, and if they're for/against it, all of which I've addressed.

Albens commented:
Updated the Safe Room airlock, after figuring out how to do the "cycle_to_external_air = 1" thing the Charon and Colony does.

*edit Also sneaks in an update to the Engine Airlock. Just making it functional. See #23854

Attempted to Squash.

Albens commented:
Updated with @ghostsheet's changes from the above PR.
Corrected the conflict, with much help from Afterthought. 
Vendors updated. This is merge ready, as I failed to specify.

SierraKomodo commented:
Be careful with putting light tubes and light bulbs on the same walls as other objects - Especially objects you're intended to click like buttons and APCs. Where the light shows up on dream maker is not where it shows up in round, and the 'Turned on light ray' effect can block clicking of objects under the light. Cameras can also cause a similar issue - They also get shifted to a different pixel position in-game. Might want to double check how they render in a compiled and run test server.

Areas I've specifically noticed this might be a problem are:
 - Starboard side of the bridge
<img width="150" alt="firefox_ZAtwHj53Yy" src="https://user-images.githubusercontent.com/11140088/57884772-4f656180-77de-11e9-822f-99ce17e09ee4.png">

 - XO's Office intercom
<img width="140" alt="firefox_noR5n9glfZ" src="https://user-images.githubusercontent.com/11140088/57884797-5d1ae700-77de-11e9-8a5a-fab9d8811f85.png">

 - COS and CE office buttons on the starboard/port sides, respectively
<img width="134" alt="firefox_vyYVVvl70q" src="https://user-images.githubusercontent.com/11140088/57884852-7f146980-77de-11e9-8ca2-6b3740b57710.png">

 - CSO office air alarm (I think its the CSO office anyway)
<img width="91" alt="firefox_XNZWtCWTCq" src="https://user-images.githubusercontent.com/11140088/57884913-a2d7af80-77de-11e9-8835-7849a167d569.png">

 - Bridge port side (Both the light and camera)
<img width="145" alt="firefox_sy2UfC2cev" src="https://user-images.githubusercontent.com/11140088/57885000-d286b780-77de-11e9-87b1-dda2773a9f1c.png">

 - SEA's office APC
<img width="86" alt="firefox_2rrlTKe5nb" src="https://user-images.githubusercontent.com/11140088/57885053-f6e29400-77de-11e9-8683-6e976c096d1d.png">

 - EVA Storage APC and air alarm
<img width="132" alt="firefox_q1I5RIVlY8" src="https://user-images.githubusercontent.com/11140088/57885117-1e396100-77df-11e9-861e-886a515eca3d.png">

 - Aft bridge airlock
<img width="118" alt="firefox_xagHUPRnWj" src="https://user-images.githubusercontent.com/11140088/57885145-2abdb980-77df-11e9-9107-b767450d5e03.png">

Chiquiii commented:
Alt clicking should negate any of those problems, but it seems like light tubes should just not radiate pixels in the first place.

SierraKomodo commented:
Requiring people to alt+click to access something that should be clickable in map is poor design IMO.

afterthought2 commented:
@SierraKomodo is correct in that having multiple wall objs on the same wall turf, esp. from the same direction, is undesirable.

Albens commented:
Thanks for the feedback.
Unfortunately, the system of pixel-shifting Lights and Cameras at roundstart is awful. Right pain in the ass to work with. It's a requirement to have multiple wall objs on the same turf, in the same dir or not. Even if Torch was larger, you'd still have objects on top of each other.

I'd vehemently disagree that putting objects in the same tile is undesirable; The system that shifts lights and Cameras is *undesirable* in the current iteration.  

With that said, I've updated ~4 or 5 objects that where shifted wrong, only 2 of them where from my edit, however. Existing Torch **full** of objects that are overlapping, and looking for them in the editor is not practical.
<details><summary>Some pictures</summary>
Objects with small issues that can be visually unappealing, but practically a non-issue.

 ![Image](https://i.gyazo.com/2a8bf2f216c6eb7a8cd8845d8de395f3.png)

 ![Image](https://i.gyazo.com/783361c235b0f7b93cdaa4889dd07dcd.png)

 ![Image](https://i.gyazo.com/60a5d8b7fdcb9d5f344fd4647ea627a2.png)

 ![Image](https://i.gyazo.com/70f806e35e2b75b06babba6781521b1a.png)

 ![Image](https://i.gyazo.com/0640a9f997ff24a32962015112125ee0.png)

Objects that come out nice.
 ![Image](https://i.gyazo.com/8fdc4449eefdc8774a5829a82436237c.png)

 ![Image](https://i.gyazo.com/73c3d3f27f596bb4491a3a5032c16a13.png)

 ![Image](https://i.gyazo.com/d3d24d54f6ff9b95e08a3ca052a42744.png)

</p>
</details>

Regardless; I've updated with fixes, and have touched up some of the existing Torch issues on B-deck. However, I'm not going out of my way to edit areas I didn't touch. That's intentional, obviously. 

*edit* Ready for Review, as previously.  I don't intend to update any further unless other issues are found.

Roland410 commented:
Lights/switches/other objects on the same wall/tile not only look bad, but can interfere when you try to click one of them. Most commonly clicking the lights instead of the button.

Albens commented:
Not ready for review; Possible issue with UMBRA still breaking. I'm not aware why UMBRA Is still breaking, so if you know why, please help. 

I forgot to credit the person who helped me originally, too..
*Edit*
@ChaosAlpha

Albens commented:
With many thanks to @ChaosAlpha the issue regarding Manuals.DM has been fixed. After testing, the UMBRA Envelope spawns correctly now. 
Again Ready for Review. 
There are several changes in here that *might* need to be explained, such as Safe Room, Bridge Arms Lockers, perhaps an office or two, and the Atmospheric station in maint. Some of them are obvious, but if anyone has any questions, hit me up on Discord.

Albens commented:
Updated with changes based off of Afterthought and Cakey's feedback, and some others.

Albens committed:
Bridge deck Commit

Commited with all squashes.
Manuals updated thanks to ChaosAlpha, many thanks for dealing with my stupid.

xales reviewed (approved):
map diff is back, might as well give it a shot with more people now
